src/vm/loaderallocator.cpp

```diff
@@ -1444,14 +1444,16 @@ BOOL AppDomainLoaderAllocator::CanUnload()
     return m_Id.GetAppDomain()->CanUnload();
 }
 
-#ifndef CROSSGEN_COMPILE
 BOOL AssemblyLoaderAllocator::CanUnload()
 {
     LIMITED_METHOD_CONTRACT;
 
+#ifdef CROSSGEN_COMPILE
+    UNREACHABLE();
```

Comment:
This ifdef is not necessary - no harm in this returning TRUE when compiled for crossgen.